### PR TITLE
Modified routes

### PR DESCRIPTION
**Pull request by VinayV9**
new route created from single point  to index.js
from index.js we we define as routes as we want

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ app.use(bodyParser.json())
 
 
 //routes
-app.use("/", require("./routes/notes"))
+app.use("/", require("./routes/index"))
```
Comment (skyshader) [inline server/app.js]: not required index in the end

Comment (VinayV9) [reply server/app.js]: will it look index.js defaulty

Comment (skyshader) [reply server/app.js]: yes it will. just mention `require('./routes')`

Comment (VinayV9) [reply server/app.js]: ok it's working i changed

**Pull request merged**